docs/pages/api-docs/data-grid/data-grid-pro.md

```diff
@@ -132,7 +132,7 @@ You can use the [slots API](/components/data-grid/components/#overriding-compone
 
 | Name | Type | Default | Description |
 |:-----|:-----|:--------|:------------|
-| <span class="prop-name">Checkbox</span> | <span class="prop-type">elementType</span> | <span class="prop-type">Checkbox</span> | Checkbox component used in the grid for both header and cells. By default, it uses the Material-UI core Checkbox component. |
+| <span class="prop-name">Checkbox</span> | <span class="prop-type">elementType</span> | <span class="prop-type">Checkbox</span> | Checkbox component used in the grid for both header and cells. By default, it uses the MUI core Checkbox component. |
```

Comment:
```suggestion
| <span class="prop-name">Checkbox</span> | <span class="prop-type">elementType</span> | <span class="prop-type">Checkbox</span> | Checkbox component used in the grid for both header and cells. By default, it uses the MUI core Checkbox component. |
```

Reply:
What is the difference between the two lines here ?

Reply:
It was the same oops. I wondered about the capitalization of the MUI Core.

Reply:
If we approved this one, could we update the whole codebase? I assume that the source of truth for this one is in the TypeScript' descriptions.

Reply:
> If we approved this one, could we update the whole codebase? I assume that the source of truth for this one is in the TypeScript' descriptions.

I think it's not worth it to waste time updating the manually authored API pages now. The documentation for this component can be automatically generated in #2465. BTW, the TypeScript definition doesn't have the "By default, it uses the MUI..." part.

Reply:
> BTW, the TypeScript definition doesn't have the "By default, it uses the MUI..." part.

Oh nice, +1 for ignoring the api .md files and focusing on the TypeScript definition. It started to diverge on this instance with #1528.